meerk40t/core/node/node.py

```diff
@@ -211,6 +212,50 @@ def parent(self):
     def root(self):
         return self._root
 
+    @property
```

Comment:
All nodes were given can_action() values for emphasize, highlight, target, move, scale, rotate, skew, modify, alter, update, remove. (These were never used). The idea was to allow checking their action and by default locked objects could do none of these things.